mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -9,6 +9,8 @@
         <GlossaryWeakArray index="1" value="Roach"/>
         <GlossaryWeakArray index="2" value="Stalker"/>
         <CardLayouts index="0">
+            <LayoutButtons index="6" Face="Cancel" AbilCmd="AdeptPhaseShiftCancel,Execute"/>
+            <LayoutButtons index="7" Face="Rally" AbilCmd="ProgressRally,Rally1"/>
```

Comment:
Fixed an issue where Adepts could not be ordered to cancel the Shade ability when selected with Adepts who are not finished warping in

The cancel button is moved above the rally button, so it receives priority to be displayed on the command card.
This will hide and disable the rally shortcut and button. You should still be able to rally with right click.

The buttons could have been moved next to each other, though